app/controllers/home_controller.rb

```diff
@@ -1,5 +1,5 @@
 class HomeController < ApplicationController
   def index
-    @company = Company.new
+    
   end
 end
```

Comment:
I think this is failing a test as well. Did the new specs remove the signup form form the home page?

![screen shot 2015-04-21 at 9 58 56 pm](https://cloud.githubusercontent.com/assets/8313881/7267042/ac876616-e871-11e4-98e8-731538f8271a.png)